lib/developer_portal/lib/liquid/filters/rails_helpers.rb

```diff
@@ -46,7 +46,10 @@ def stylesheet_link_tag(name)
       def javascript_include_tag(name)
         js = RailsHelpers.replace_googleapis(name)
 
-        if js != name || THREESCALE_JAVASCRIPTS.include?(js)
+        case
+        when name == 'stats.js'
```

Comment:
Why?

Reply:
So, this is quite a hack! basically we had a rake task in the past for generating the developer portal `stats.js`. With @hallelujah we thought it would be good right now to have this step to automatically compile and include this asset without touching the partial and thus avoid updating it for everyone then. This is only a stage, the final one will be crafting a new filter and updating the templates. WDYT?

Reply:
👍 maybe worth a comment in the code